docs/readmes/orc8r/upgrade_1_4.md

```diff
@@ -0,0 +1,66 @@
+---
+id: upgrade_1_4
+title: Upgrade to v1.4
+hide_title: true
+---
+
+# Upgrade to v1.4
+
+This guide covers upgrading Orchestrator deployments from v1.3 to v1.4.
+Upgrades which skip versions (e.g. v1.2 to v1.4) are not explicitly documented
+or supported at this time.
+
+The v1.4 upgrade follows a standard procedure:
+
+- Upgrade the Terraform modules
+- Bump version numbers
+
+## Prerequisites
+
+Build and publish the application containers and helm charts on the head of the
+release branch by following the documentation [here](https://docs.magmacore.org/docs/orc8r/deploy_build).
+
+If you are using local Terraform state (the default), ensure all Terraform state files (i.e. [`terraform.tfstate`](https://www.terraform.io/docs/state/index.html)) are located in your working directory before proceeding. This means `terraform show` should list existing state (rather than outputting `No state`).
+
+## Upgrade Terraform Modules
+
+Set the `source` values for each of the Orchestrator modules in your root
+Terraform module to point to this release's modules and bump chart and
+container versions:
+
+```hcl-terraform
+module orc8r {
+  source = "github.com/magma/magma//orc8r/cloud/deploy/terraform/orc8r-aws?ref=v1.4"
+  # ...
+}
+
+module orc8r-app {
+  source = "github.com/magma/magma//orc8r/cloud/deploy/terraform/orc8r-helm-aws?ref=v1.4"
+  # ...
+  orc8r_chart_version   = "1.5.7
+  orc8r_tag             = "MAGMA_TAG"       # from build step, e.g. v1.4.0
+  orc8r_deployment_type = <deployment_type> # valid options are: ['fwa', 'federated_fwa', 'all']
```

Comment:
- Nit: same as above, prefer `DEPLOYMENT_TYPE` over `<deployment_type>`
- Options should be enclosed in double quotes (`"fwa"`) rather than single